ci/environment.yml

```diff
@@ -4,9 +4,13 @@ channels:
 dependencies:
   - dask
   - netcdf4
+  - numba
+  # pinning the numpy version for numba compatibility
+  - numpy>=1.2, <1.22
```

Comment:
@anissa111 do we need this? or just requirements is enough.

Reply:
@rajeeja I think we need to have it in the environment file as well to keep the pinned version in our local development environment